### PR TITLE
[Feature][AUT-2048] Enable text to wrap pictures

### PR DESCRIPTION
**Pull request by shpran**
# [AUT-2048](https://oat-sa.atlassian.net/browse/AUT-2048)

## Require PRs
* https://github.com/oat-sa/tao-core-ui-fe/pull/446

## Changes
* Added migration to remove `Image Alignment` status flag
* `SetImageAligmentConfig` installation script was changed:
  * Will enable `Image Alignment` plugin by default
  * Added public constant `PARAM_MEDIA_ALIGNMENT` for `mediaAlignment` parameter
* `SetImageAligmentConfig` installation script was removed from manifest file, as we do not need this flag anymore

## How to test
* Installation
  1. Do a fresh instance installation
  2. Check that `Image Alignment` plugin is enabled
* Update
  1. Check that `Image Alignment` plugin is disabled
  2. Update an existing instance
  3. Check that `Image Alignment` plugin is enabled

### CODE REVIEW

**Review by pnal — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Review by hectoras — APPROVED** [review]

File: views/package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
         "@oat-sa/tao-core-libs": "0.4.5",
         "@oat-sa/tao-core-sdk": "1.18.0",
         "@oat-sa/tao-core-shared-libs": "1.2.0",
-        "@oat-sa/tao-core-ui": "1.53.1",
+        "@oat-sa/tao-core-ui": "github:oat-sa/tao-core-ui-fe#feature/aut-2048/Remove-alignment-flag",
```
Comment (shpran) [inline views/package.json:20]: - [ ] Check before merge

File: scripts/install/SetImageAligmentConfig.php [hunk 2]
```diff
@@ -15,40 +15,46 @@
...
+        $mediaAlignment = filter_var(
+            $params[self::PARAM_MEDIA_ALIGNMENT] ?? true,
+            FILTER_VALIDATE_BOOLEAN
+        );
+
+        $this->getClientLibConfigRegistry()->register(
             'ui/image/ImgStateActive',
             [
-                'mediaAlignment' => $mediaAligment
+                'mediaAlignment' => $mediaAlignment,
```
[the remark this answers is not in the document]
Comment (gabrielfs7) [reply scripts/install/SetImageAligmentConfig.php:49]: Ok ok, thanks for clarifying. Maybe lets just deprecate for now :)

**Review by gabrielfs7 — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Review by jsconan — APPROVED** [review]
Looks good 👍 

**Review Checklist**
- ~~[ ] New code is covered by tests (if applicable)~~
- ~~[ ] Tests are running successfully (old and new ones) on my local machine (if applicable)~~
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [x] Commits are following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] Commits messages are meaningful
- [x] Commits are atomic

**Pull request merged**